CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- [File I/O Tracking](https://docs.sentry.io/platforms/apple/performance/instrumentation/automatic-instrumentation/#file-io-tracking) GA (#2212)
```

Comment:
Does everyone know what "GA" means? I had to google it. Why not write it out, or just say "enabled by default" or something like that?